docs/source/roadmap.md

```diff
@@ -147,23 +76,20 @@ Nevertheless, to demonstrate _starfish_'s scale, we will implement workflows on
 ### A Developer Focus Point
 Two key aims of _starfish_ and SpaceTx are to (1) encourage convergence of the community onto a single object model for processing of image-based transcriptomics data and (2) make it easier to develop pipelines by providing a high-performance platform. To facilitate this, we must encourage developers to work on _starfish_ through active outreach and by being encouraging and helpful.
 
-## Outstanding work to complete 0.1.0
+## Near-term goals
```

Comment:
This confused me (I assumed this was pre-hackathon). I think we need to be more specific about what this entails. Outstanding work is a good description but I'm fine if you want to re-word it. 

It may also be clearer to write a short paragraph that elaborates.

Reply:
I wanted to keep your long-form descriptions of the MVP, which is now split across two releases: 0.1.0 and 0.2.0

Reply:
reverted to "Outstanding work"